### DISCUSSION TITLE
Globalize support?

### DISCUSSION THREAD

**Discussion by mbajur** [jrochkind/attr_json#232] [Ideas]
I was thinking - has anyone had any success with integrating this gem with https://github.com/globalize/globalize ? Preferably using `container` backend but i'm really open to anything. In my personal case, i would love to see it working with nested collections but let's start with the basics.

**Comment by jrochkind** [comment]
Hi thanks for your note!

I'm the maintainer here, but I haven't done too much with globalize.  I'm not sure if I totally understand in what ways you are wanting to integrate it?  Since this is already based on ActiveModel and ActiveRecord::Attributes, I'd think it would already work well?

To be honest I probably won't have any time to work on this, but if you found something was necessary here that could be accomplished with fairly minimal maintainable code, I'd be happy to review a PR!

The first step if someone wanted to work on it, I think, would be providing code examples that you would want to work, that don't work how you would want, and explaining what you'd want them to do that it doesn't currently do!

**Reply by mbajur** [reply]
How i picture it is basically how you define globalize configs for your models in the default attr_json-less setup. So, something like:

```
class MyModel
  translates :description

  class Sections
    include AttrJson::Model
    
    attr_json :title

    translates :title
  end
end
```

And then, when rendering that:

```ruby
MyModel.last.sections[0].title # => Default value
MyModel.last.sections[0].title_en # => EN translation
```

But that's super mind-bending to even start to think how that could be possibly done
